static/js/settings_toggle.js

```diff
@@ -31,6 +31,40 @@ exports.initialize = function () {
     $("#settings_overlay_container .tab-container").append(toggler.get());
 };
 
+// Handles the collapse/reveal of some tabs in the org settings for non-admins.
+exports.toggle_org_setting_collapse = function (elem) {
+    var is_collapsed = $(".collapse-org-settings").hasClass("hide-org-settings");
+
+    if (is_collapsed) {
+        elem.each(function () {
+            var show_fewer_settings_text = i18n.t("Show fewer");
+
+            $("#toggle_collapse_chevron").removeClass("fa-angle-double-down");
+            $("#toggle_collapse_chevron").addClass("fa-angle-double-up");
+
+            $(this).removeClass("hide-org-settings");
+            $("#toggle_collapse").text(show_fewer_settings_text);
+
+        });
+    } else {
+        elem.each(function () {
+            var show_more_settings_text = i18n.t("Show more");
+
+            $("#toggle_collapse_chevron").removeClass("fa-angle-double-up");
+            $("#toggle_collapse_chevron").addClass("fa-angle-double-down");
+
+            $(this).addClass("hide-org-settings");
```

Comment:
Everything but this line should be outside the loop, right?

Reply:
Yes, this was an oversight. Thanks for pointing it out!